ceph-ansible-pull-requests/config/definitions/ceph-ansible-pull-requests.yml

```diff
@@ -59,3 +59,11 @@
           !include-raw:
             - ../../../scripts/build_utils.sh
             - ../../build/build
+
+    publishers:
+      - github-notifier
+      - postbuildscript:
+          script-only-if-succeeded: False
```

Comment:
Don't we want to do this regardless of if the job succeeds or fails? The way it is now it'd only happen on failure.